common/bytes.go

```diff
@@ -23,29 +23,38 @@ import (
 	"encoding/hex"
 	"fmt"
 	"math/big"
+	"regexp"
 	"strings"
+
+	"github.com/ethereum/go-ethereum/logger"
+	"github.com/ethereum/go-ethereum/logger/glog"
 )
 
-func ToHex(b []byte) string {
-	hex := Bytes2Hex(b)
-	// Prefer output of "0x0" instead of "0x"
-	if len(hex) == 0 {
-		hex = "0"
+// BytesToHex converts a byte slice to hexadecimal notation, prefixed with 0x.
+func BytesToHex(blob []byte) string {
+	return "0x" + hex.EncodeToString(blob)
+}
```

Comment:
Can we name this `Bytes0x`? I think it needs to be obvious in the name what the difference to hex.EncodeToString is.